packages/components/src/LightBox/LightBox.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import ExternalLightBox from "react-image-lightbox";
+
+// Library requires fetching it's CSS.
```

Comment:
I may have written this but… 😉 

```suggestion
// Library requires fetching its CSS.
```